src/btree/bt_split.c

```diff
@@ -499,7 +499,7 @@ __split_root(WT_SESSION_IMPL *session, WT_PAGE *root)
     /*
      * Flush our writes and start making real changes to the tree, errors are fatal.
```

Comment:
Q: Would it make sense to say "Make sure previous writes to memory have **released**" instead of completed? It's technically true and matches the new macro name, however we're touching on low levels details that "completed" can gloss over.

Reply:
This one falls into the bucket of, "I don't really know what this comment was trying to say or achieve, if there is any ordering here I'll have to spend hours looking for it." If we were to straight up remove the "flush" phrase I think we'd need to do a full investigation as to whether this barrier does anything. Which is why I opted to not modify this comment at all.

Reply:
Fair point. As discussed we'll move the flush changes into [WT-10967](https://jira.mongodb.org/browse/WT-10967)